buildSrc/src/main/java/xyz/wagyourtail/doclet/tsdoclet/Main.java

```diff
@@ -25,6 +25,12 @@ public class Main implements Doclet {
     public static FileHandler outputTS;
     public static PackageTree classes = new PackageTree("_javatypes");
     public static DocTrees treeUtils;
+    public static Set<String> redirectNeeded = new HashSet<>();
+
+    public static final Map<String, String> missingExtends = new HashMap<>() {{ // expand needed
+        put("ScriptScreen", "IScreen");
+        put("NBTElementHelper", "NBTElementHelper$NBTCompoundHelper & NBTElementHelper$NBTListHelper & NBTElementHelper$NBTNumberHelper");
+    }};
```

Comment:
can probably do this with an annotation, like with `@DocletReplaceParams`, also technically IScreen is added to all minecraft screens, so this can be on the `BaseScreen` jsmacros class.

the nbt one I don't like, users should have to check the subtype to get access to the subtype's functions

Reply:
didn't know there's a convenient thing called `@DocletReplaceParams`!
i regret about not asking protip at the beginning

Reply:
added in 2b63517